fe/fe-core/src/main/java/org/apache/doris/qe/ShowExecutor.java

```diff
@@ -1653,6 +1656,20 @@ private void handleShowAlter() throws AnalysisException {
         resultSet = new ShowResultSet(showStmt.getMetaData(), rows);
     }
 
+    // Show character set.
+    private void handleShowCharset() throws AnalysisException {
+        ShowCharsetStmt showStmt = (ShowCharsetStmt) stmt;
+        List<List<String>> rows = Lists.newArrayList();
+        List<String> row = Lists.newArrayList();
+        // | utf8mb4 | UTF-8 Unicode | utf8mb4_general_ci | 4|
+        row.add("utf8mb4");
```

Comment:
better use value define in `SessionVariable`, such as collationConnection, charsetResults, charsetResults, collationDatabase, etc. You can get it from ctx.getSessionVariables().getxxx.

And please add ut